packages/story-editor/src/app/highlights/quickActions/test/useQuickActions.js

```diff
@@ -93,8 +93,13 @@ jest.mock('@googleforcreators/media', () => ({
   },
 }));
 
+jest.mock('../../../media', () => ({
+  ...jest.requireActual('../../../media'),
+  useLocalMedia: jest.fn(),
+  TRANSCODABLE_MIME_TYPES: ['video/mp4'],
```

Comment:
This is not needed for the tests to work.

`useQuickActions` only uses `TRANSCODABLE_MIME_TYPES` this to check whether `optimizeVideo` needs to be called first.

The tests only check whether `postProcessingResource` is ultimately called.

Reply:
@timarney I meant **only** line 99 is not needed; not the whole mock. Now the tests are failing again.